ui/package.json

```diff
@@ -10,7 +10,8 @@
     "lint": "npm run lint --workspaces",
     "lint:fix": "npm run lint:fix --workspaces",
     "start": "npm run start -w app",
-    "clean": "rimraf .build-cache && npm run clean --workspaces"
+    "clean": "rimraf .build-cache && npm run clean --workspaces",
+    "reinstall": "npm run clean && rm -rf **/node_modules && npm install"
```

Comment:
Are we sure we want to remove the `node_modules`, looks like pretty heavy to do it.
Just questioning if it's really intended, I have nothing against it :)

Reply:
ah yeah ok I re read the new comments in the UI and it is the intention.